src/app/app.module.ts

```diff
@@ -166,6 +167,7 @@ export function provideAppBridgeService(http) {
     FormsModule,
     HttpModule,
     ReactiveFormsModule,
+    ScrollDispatchModule,
```

Comment:
If you add an `exports: [ ScrollDispatchModule ]` then elements that import NovoElements will just get it for free